modern-business/style-rtl.css

```diff
@@ -4649,17 +4649,29 @@ footer .wp-block-a8c-navigation-menu .sub-menu {
 }
 
 .entry .entry-content .wp-block-cover-image h2.has-text-align-left,
-.entry .entry-content .wp-block-cover h2.has-text-align-left {
+.entry .entry-content .wp-block-cover h2.has-text-align-left,
```

Comment:
These changes were autogenerated when I ran `npm run build`, probably just some other stuff that didn't build properly in a different PR.